### ISSUE TITLE
Not finding GIT folder - No source control providers registered

### ISSUE THREAD

**Issue by brannonvann** [microsoft/vscode-remote-release#4042]
- VSCode Version: 1.51.1 & 1.52.0-insider
- Local OS Version: MacOs 10.15.7
- Remote OS Version: Ubuntu 16.04.7 LTS
- Remote Extension/Connection Type: SSH

Steps to Reproduce:

1. Install as documented in the [docs](https://code.visualstudio.com/docs/remote/ssh)
2. Connect to remote machine
3. Open a [project] folder that contains a .git folder on the remote machine
4. Observe "No source control providers registered." in the Git window in Code.
 
Does this issue occur when you try this locally?: No
Does this issue occur when you try this locally and all extensions are disabled?: n/a

I have tried the suggestions found in tickets: [#1143](https://github.com/microsoft/vscode-remote-release/issues/1443) & [#1848](https://github.com/microsoft/vscode-remote-release/issues/1848) and I posted to stackoverflow but the more I research the more I think this might be a bug so i'm reporting it here. I can provide whatever logs/output might be necessary. 

I'm suspecting this might be a problem with the install or extensions. I'm not sure if this is true but I can't seem to get any of the third party extensions to work such as prettier. Global search also seem to not function. My main concern is GIT but thought I would mention this in case it helps.

Additionally, git works from the terminal on my remote machine as expected. I also don't have an issue when I share the folder and attach to it and load locally on my machine. 

![image](https://user-images.githubusercontent.com/40714964/99553617-56999c80-298c-11eb-9b1a-1a1320f95c6c.png)

**Comment by rakannimer** [comment]
Having the same issue, it also happens in insiders version and in workspaces.

**Comment by rakannimer** [comment]
Looks like it's working again for me 👍

**Comment by brannonvann** [comment]
It still broken for me. @rakannimer What did you do to resolve the issue?

**Comment by rakannimer** [comment]
Didn't do anything, I tried re-opening the same workspace and it worked. Not sure what changed sorry !

**Comment by brannonvann** [comment]
Does anyone know if this is because I'm running ubuntu 16.04 on a ARMv7?

**Comment by brannonvann** [comment]
@eamodio I noticed that you are the creator of GitLens which I use so I thought i would try to install that on the remote machine but it also doesn't seem to work. I noticed that when I open a new folder there is a "Activating Extensions" in the bottom bar of VS Code. When open the associated window it lists several "Activating Extension"s but  it quickly terminates. It's as if none of the extensions never actually activate. 

As a side note, thank you for creating GITLens it's very helpful!
